Sourcery.xcodeproj/project.pbxproj

```diff
@@ -1153,6 +1153,7 @@
 			developmentRegion = English;
 			hasScannedForEncodings = 0;
 			knownRegions = (
+				English,
```

Comment:
I don't think this has an impact but this could be removed I think 🤔

Reply:
Done.